spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -719,6 +714,12 @@ def run(self, run_time):
                 self._do_load()
                 loading_done = True
 
+        # Disable auto pause and resume if the binary can't do it
+        if (self._executable_start_type !=
+                ExecutableStartType.USES_SIMULATION_INTERFACE):
+            self._config.set("Buffers", "use_auto_pause_and_resume",
+                             "False")
```

Comment:
alan says: if its where it originally was, then itll never be set, as the executable start type is now only set at load time. 

theres a equiv of thiswhich had to have a is not virtual board due to the same interaction with multirun